core/src/main/scripts/importer/allowed_data_types.txt

```diff
@@ -32,3 +32,4 @@ METHYLATION	CONTINUOUS	methylation_hm450
 FUSION	FUSION	fusion
 PROTEIN_LEVEL	LOG2-VALUE	rppa
 PROTEIN_LEVEL	Z-SCORE	rppa_Zscores
+PROTEIN_LEVEL	Z-SCORE	ms_abundance
```

Comment:
Next to Z-SCORE, we will need another one here for normal protein levels, similar to LOG2-VALUE rppa on row 33 above.